www/src/context/CommissioningPackageContext.tsx

```diff
@@ -9,40 +9,76 @@ export interface CommissioningPackageContextProps {
   setCommissioningPackages: React.Dispatch<
     React.SetStateAction<CommissioningPackage[]>
   >;
+  deleteCommissioningPackage: (packageId: string) => void;
 }
 
 const CommissioningPackageContext = createContext<
   CommissioningPackageContextProps | undefined
 >(undefined);
 
+const createInitialPackage = (): CommissioningPackage => ({
+  id: "asset:Package1",
+  name: "Initial Package",
+  color: HighlightColors.LASER_LEMON,
+  boundaryIds: [],
+  internalIds: [],
+  nodeIds: [],
+});
+
 export const CommissioningPackageContextProvider: React.FC<{
   children: React.ReactNode;
 }> = ({ children }) => {
-  const [activePackage, setActivePackage] = useState<CommissioningPackage>({
-    id: "asset:Package1",
-    name: "Initial Package",
-    color: HighlightColors.LASER_LEMON,
-    boundaryIds: [],
-    internalIds: [],
-    nodeIds: [],
-  });
-  const [commissioningPackages, setCommissioningPackages] = useState<
-    CommissioningPackage[]
-  >([]);
+  const [activePackage, setActivePackage] = useState<CommissioningPackage>(createInitialPackage());
+  const [commissioningPackages, setCommissioningPackages] = useState<CommissioningPackage[]>([]);
 
   useEffect(() => {
     if (activePackage && commissioningPackages.length === 0) {
       setCommissioningPackages([activePackage]);
     }
   }, [activePackage, commissioningPackages]);
 
+  const deleteCommissioningPackage = (packageId: string) => {
```

Comment:
Dette ser veldig bra ut! Grunnen til at de slettede pakkene dukker opp igjen er fordi de ikke blir slettet fra databasen. Må gjøre kall for å slette alle noder/boundaries i en pakke. Hvis ikke vil frontenden hente nodene som allerede eksisterer i databasen når en identisk pakke lages.

Reply:
Ahh det gir mening, takk! Har lagt til kall til databasen nå for å slette!